src/__tests__/utils/feedbackGif.test.ts

```diff
@@ -0,0 +1,22 @@
+import { getRandomGifSrc } from '../../utils/feedbackGif';
+
+const arr = [0, 1, 3];
+let isCorrect = true;
+describe('randomly generating the correct/incorrect gifs:', () => {
+  it('from the correct-gif & its path', () => {
```

Comment:
I think it's best to check what the return value `startsWith()` in order to see if the `/feedbackGifs/correct/` or `/feedbackGifs/incorrect/` is the right one.

and then leave a single test with the `.toContain()` structure you have currently